py3status/modules/file_status.py

```diff
@@ -1,22 +1,45 @@
 # -*- coding: utf-8 -*-
 """
-Display if a file or directory exists.
+Display if a files or directories exists.
 
 Configuration parameters:
     cache_timeout: how often to run the check (default 10)
-    format: format of the output. (default '{icon}')
-    icon_available: icon to display when available (default '●')
-    icon_unavailable: icon to display when unavailable (default '■')
-    path: the path to a file or dir to check if it exists (default None)
+    format: format of the output. (default '\?color=paths [\?if=paths ●|■]')
+    format_path: format of the path output. (default '{basename}')
+    format_path_separator: show separator if more than one (default ' ')
+    path: the path(s) to a file or dir to check if it exists, take a list (default None)
+    thresholds: specify color thresholds to use (default [(0, 'bad'), (1, 'good')])
 
 Color options:
     color_bad: Error or file/directory does not exist
     color_good: File or directory exists
 
 Format placeholders:
-    {icon} icon for the current availability
-
-@author obb, Moritz Lüdecke
+    {format_path} paths of matching files
+    {paths} number of paths, eg 1, 2, 3
+
+format_path path placeholders:
+    {basename} basename of matching files
+    {fullpath} fullpath of matching files
+
+Examples:
+```
+# check files with wildcard, or contain user path, full paths
+file_status {
+    path = ['/tmp/test*', '~user/test1']
+    format = u'\?if=paths ● {format_path}|■ no files found'
+    format_path = '{fullpath}'
+}
+
+# change color on files match
+file_status {
+    path = ['/tmp/test*', '~user/test1']
+    format = u'\?color=paths ●'
+    thresholds = [(0, 'bad'), (1, 'good')]
+}
+```
+
+@author obb, Moritz Lüdecke, Cyril Levis (@cyrinux), @lasers
```

Comment:
Hmm. Thank you, but you can keep me out of this. I'm just helping you, bro. Thank you, bro. :fist_oncoming: